ballerina/build.gradle

```diff
@@ -0,0 +1,322 @@
+    }
+    externalJars {
+        transitive false
+    }
+    distribution {
+        canBeConsumed true
+        canBeResolved false
+    }
+}
+
+dependencies {
+    nativeJar project(':zipkin-extension-native')
+
+    externalJars "io.zipkin.zipkin2:zipkin:${zipkinVersion}"
+    externalJars "io.zipkin.reporter2:zipkin-reporter:${zipkinReporterVersion}"
+    externalJars "io.zipkin.reporter2:zipkin-sender-okhttp3:${zipkinReporterVersion}"
+    externalJars "com.squareup.okio:okio:${okioVersion}"
+    externalJars "com.squareup.okhttp3:okhttp:${okhttpVersion}"
+    externalJars "org.jetbrains.kotlin:kotlin-runtime:${kotlinVersion}"
+    externalJars "org.jetbrains.kotlin:kotlin-stdlib:${kotlinStdlibVersion}"
+    externalJars "io.opentelemetry:opentelemetry-exporter-zipkin:${openTelemetryVersion}"
+    externalJars "io.opentelemetry:opentelemetry-api:${openTelemetryVersion}"
+    externalJars "io.opentelemetry:opentelemetry-api-metrics:${openTelemetryAlphaVersion}"
+    externalJars "io.opentelemetry:opentelemetry-context:${openTelemetryVersion}"
+    externalJars "io.opentelemetry:opentelemetry-sdk-trace:${openTelemetryVersion}"
+    externalJars "io.opentelemetry:opentelemetry-sdk-common:${openTelemetryVersion}"
+    externalJars "io.opentelemetry:opentelemetry-semconv:${openTelemetryAlphaVersion}"
+    externalJars "io.opentelemetry:opentelemetry-proto:${openTelemetryAlphaVersion}"
+    externalJars "io.opentelemetry:opentelemetry-exporter-otlp-trace:${openTelemetryVersion}"
+    externalJars "io.opentelemetry:opentelemetry-exporter-otlp-common:${openTelemetryVersion}"
+    externalJars "io.opentelemetry:opentelemetry-extension-trace-propagators:${openTelemetryVersion}"
+    externalJars "com.google.guava:guava:${guavaVersion}"
+    externalJars "com.google.guava:failureaccess:${failureAccessVersion}"
+}
+
+clean {
+    delete "${project.projectDir}/target"
+    delete "${project.projectDir}/Ballerina.lock"
+    delete "${project.projectDir}/ballerina-internal.log"
+}
+
+jar {
+    manifest {
+        attributes('Implementation-Title': project.name, 'Implementation-Version': project.version)
+    }
+}
+
+task unpackJballerinaTools(type: Copy, dependsOn: configurations.jbalTools) {
+    configurations.jbalTools.resolvedConfiguration.resolvedArtifacts.each { artifact ->
+        from zipTree(artifact.getFile())
+        into new File("${buildDir}/extracted-distributions", "jballerina-tools-zip")
+    }
+}
+
+task unpackStdLibs(dependsOn: configurations.ballerinaStdLibs) {
+    doLast {
+        configurations.ballerinaStdLibs.resolvedConfiguration.resolvedArtifacts.each { artifact ->
+            copy {
+                from project.zipTree(artifact.getFile())
+                into new File("${buildDir}/extracted-distributions", artifact.name + "-zip")
+            }
+        }
+    }
+}
+
+task copyExternalJarsToLib(type: Copy) {
+    into "${project.projectDir}/lib"
+    from configurations.externalJars
+}
+
+task copyStdlibs(type: Copy) {
```

Comment:
Shall we directly use final distribution instead of jballerina intermediate pack?

Reply:
@keizer619 addressed the suggestions.